Roles/Setup/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# tasks file for Setup
+
+  - name: Install pip from apt
+    apt: name=python-pip state=present
+
+  - name: Update APT package cache
+    apt: update_cache=yes
+
+  - name: Run apt-get upgrade
+    apt: upgrade=full
+
+  - name: Username and Password
+      user: user
+      password: "{{password}}"
```

Comment:
vasika koita na deis... exw valei sto arxeio me ta hosts : /Project/host_vars gia kathe node mesa ena hashed password. thelw me username user kathe fora na zhtaei kwdiko kai na mporei na mpei o xrhsths mesa.. twra omws edw auto uparxei sto Project/Setup/tasks/main.yml kai oi kwdikoi gia kathe ena sto Project/host_vars/node_(tade). pws tha to kanoume?